cross/libvpx/Makefile

```diff
@@ -46,6 +46,60 @@ ifeq ($(findstring $(ARCH),$(PPC_ARCHES)),$(ARCH))
 CONFIGURE_ARGS += --target=generic-gnu
 endif
 
+# Removal of GCC additional warning flags required at least for ppc853x-5.2 qoriq-5.2
```

Comment:
If I understood well the idea of @GuillaumeSmaha, enabling a patch that way documents which arch is concerned.